### PR TITLE
TCR

### PR DESCRIPTION
**Pull request by ZongpengYang**

### CODE REVIEW

File: linux_sql/README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
 # Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+
+## Introduction
+Cluster Monitor Agent is an internal tool that monitors the cluster resources and save the collected information into a database for furture use. This helps the infrastrue team to monitor the efficiency and hardware status of each server. In case that something went wrong, the infrasture team can inspect the usage information collected to indentify the problem.
+
+## Directory shortcut
```
Comment (Jarvis-deprecated-account) [inline linux_sql/README.md]: remove

File: linux_sql/scripts/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	exit 1
+# check if the password is entered with start
+elif [[ "$1" == "start" && "$#" == 1 ]];
+then
+	echo "WARNING: You did not enter password..." >&2
+	exit 1
+fi
+
+# check if it is start or stop
+if [ "$1" == "start" ];
```
Comment (Jarvis-deprecated-account) [inline linux_sql/scripts/psql_docker.sh]: set args

File: linux_sql/scripts/psql_docker.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	else
+		docker start jrvs-psql
+		exit 0
+	fi
+# check if it is stop 
+elif [[ "$1" == "stop" && "$#" == 1 ]];
+then
+	#stop the container & the docker
+	systemctl status docker && docker stop jrvs-psql
+	systemctl status docker && systemctl stop docker
```
Comment (Jarvis-deprecated-account) [inline linux_sql/scripts/psql_docker.sh]: don't stop docker server

File: linux_sql/sql/queries.sql [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+SELECT 
+  cpu_number, id AS host_id, total_mem
+FROM host_info
+ORDER BY
+  cpu_number ASC,
+  total_mem DESC;
+
+
+--Q2--
+--create temporary table to sort calculated values
```
Comment (Jarvis-deprecated-account) [inline linux_sql/sql/queries.sql:11]: combine everything

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**